js/src/collect/collect-v5.js

```diff
@@ -10,13 +13,15 @@ module.exports = function() {
 	var outerFields = [];
 	var acfFields = [];
 
-	if ( wp.data.select( "core/block-editor" ) ) {
+	// Check whether classic editor is used.
+	if ( isTinyMCEAvailable( "content" ) ) {
+		acfFields = acf.get_fields();
+	} else {
+		// Assume Gutenberg is used.
```

Comment:
As discussed in [this thread](https://yoast.slack.com/archives/C03PRESAXMJ/p1674826220909549), we already have a reliable method to know whether the editor is Gutenberg or not, by accessing `window.wpseoScriptData.isBlockEditor`. If we can use this in this check, it means that we wouldn't need to duplicate the helper logic for tinyMCE.

Reply:
Also, in this file` js/src/collect/collect-v5.js`, we have another check that uses `wp.data.select( "core/block-editor" )` (line 44). I think it's safer that we also change that check :)

Reply:
> As discussed in [this thread](https://yoast.slack.com/archives/C03PRESAXMJ/p1674826220909549), we already have a reliable method to know whether the editor is Gutenberg or not, by accessing window.wpseoScriptData.isBlockEditor. If we can use this in this check, it means that we wouldn't need to duplicate the helper logic for tinyMCE.

Agreed! I adapted it. We do not have to reset the variable in yoast-acf-analysis

Reply:
> Also, in this file js/src/collect/collect-v5.js, we have another check that uses wp.data.select( "core/block-editor" ) (line 44). I think it's safer that we also change that check :)

Disagreed: unlike the check that we changed, this check is not primarily aimed ad checking which editor is active. It primarily checks if the `core/block-editor` is available (not undefined) so that it can safely call `var blocks = wp.data.select( "core/block-editor" ).getBlocks();` a few lines later.